hoomd/md/methods/rattle.py

```diff
@@ -138,6 +138,67 @@ def _attach(self):
         super()._attach()
 
 
+class DisplacementCapped(NVE):
+    r"""NVE-like integration with capped displacement.
```

Comment:
I would describe this as "Newtonian dynamics with a cap on the maximum displacement per time step".